dict.fr.css

```
@@ -89,3 +89,11 @@ a.action.load-more::before {
 .filter-wrapper .filter:first-child::before {
   content: "Filtres";
 }
+
+.filter .products button::before {
+  content: "Products";
```

```suggestion
  content: "Produits";
```

Let's translate it immediately ;)

---

added Thanks